distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1008,11 +1009,17 @@ type Table
     @on Widget_Helpers.make_join_condition_selector
     join : Table -> Join_Kind -> Vector (Join_Condition | Text) | Text -> Text -> Problem_Behavior -> Table
     join self right join_kind=Join_Kind.Left_Outer on=(default_join_condition self join_kind) right_prefix="Right " on_problems=Report_Warning =
```

Comment:
We may add typechecks (also to in-memory) while revisiting this function:
```
    join self right (join_kind : Join_Kind = Join_Kind.Left_Outer) on=(default_join_condition self join_kind) (right_prefix:Text="Right ") (on_problems:Problem_Behavior=Report_Warning) =

Reply:
Done.